plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1504,6 +1507,43 @@ public void dropTable(ConnectorSession session, ConnectorTableHandle tableHandle
         metastore.dropTable(session, handle.getSchemaName(), handle.getTableName());
     }
 
+    @Override
+    public void renameTable(ConnectorSession session, ConnectorTableHandle tableHandle, SchemaTableName newTable)
```

Comment:
Let's keep the names in line with what's in `ConnectorMetadata`
```suggestion
    public void renameTable(ConnectorSession session, ConnectorTableHandle tableHandle, SchemaTableName newTableName)
```